pylint_odoo/checkers/no_modules.py

```diff
@@ -232,6 +240,20 @@ def visit_callfunc(self, node):
                     self.add_message('translation-field',
                                      node=argument_aux)
 
+        # Check if cr.execute() has an sql-injection
+        if node.as_string().lower().startswith('cr.execute(') or \
+                "cr.execute(" in node.as_string():
+            if len(node.args) is 1:
+                arg_string = node.args[0].as_string()
```

Comment:
Check the code of follow checks
E1306: Not enough arguments for format string
E1206: Not enough arguments for logging format string

You can see more similar checks here
http://pylint-messages.wikidot.com/all-messages

To check the original code you can see:
`import pylint;print pylint.__file__`
And review checks folder and search a code.